layout/_macro/post.swig

```diff
@@ -332,6 +332,30 @@
         {{ post.content }}
       {% endif  %}
     </div>
+
```

Comment:
So many for now. oO Mb move to external `swig` file?

Reply:
@ivan-nginx OK, seems now it need to be move to the single file.